src/common/middlewares/csrf.middleware.js

```
@@ -29,66 +29,59 @@ export const csrfProtection = (options = {}) => {
+      logger.debug(`CSRF token endpoint accessed, bypassing protection`);
+      return next();
     }
-    
-    // Always create the token from the secret
+
     try {
-      const token = tokens.create(req.session.csrfSecret);
+      // Check for token in various places
+      const token = 
+        req.headers['x-csrf-token'] || 
+        (req.body && req.body._csrf) || 
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)